api/handlers.go

```diff
@@ -119,19 +119,33 @@ func (api *API) serverRemovePOST(w http.ResponseWriter, req *http.Request, _ htt
 		api.WriteError(w, errors.New("no server found in request body"), http.StatusBadRequest)
 		return
 	}
+	ctx := req.Context()
+	// Schedule a scan for underpinned skylinks in an hour (unless one is
+	// already pending), so all of them can be repinned ASAP but also all
+	// servers in the cluster will have enough time to get the memo for the scan.
+	t := time.Now().UTC().Add(time.Hour)
+	t0, err := conf.NextScan(ctx, api.staticDB, api.staticLogger)
+	// We just set it when we encounter an error because we can get such an
+	// error in two cases - there is no next scan scheduled or there is a
+	// problem with the DB. In the first case we want to schedule one and in the
+	// second we'll get the error again with the next operation.
+	if err != nil || t0.After(t) {
+		err = conf.SetNextScan(ctx, api.staticDB, t)
```

Comment:
If you use `err = errors.Compose(err, conf.SetNextScan(...)` here you won't hide the previous error in case it's not nil.